src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -151,7 +151,7 @@ class IOUAmountPage extends React.Component {
 
     render() {
         return (
-            <View style={[styles.flex1, styles.pageWrapper]}>
+            <>
```

Comment:
I think this view is useless now?

Reply:
Yes! It's on the `AnimatedStep` that wraps the `IOUAmountPage` now.

Reply:
Can you remove `<>` in that case?

Reply:
I don't think we can. Since we removed the `View` that was wrapping the whole component, we have to wrap it in something else (we can't return multiple childrens). The React.Fragment (<>) is default way to do this. 

Let me know if you think I'm missing something.

Reply:
No, it looked to me like the outer fragment was wrapping a single component. I see now there are multiple, so no changes required.